src/client/utils/browser.js

```diff
@@ -22,40 +22,56 @@ function calculateBrowserAndVersion (userAgent) {
     };
 }
 
+
+export var isIE11;
```

Comment:
It looks like it is used for the tests. I propose to pass window through the constructor